server/chat-commands/core.ts

```diff
@@ -1583,6 +1583,11 @@ export const commands: ChatCommands = {
 			this.errorReply(`Could not find help for '/${target}' - displaying help for '/${currentBestHelp.for.join(' ')}' instead`);
```

Comment:
Oh, could you remove `typeof nextNamespace === 'boolean'` on line 1563? It no longer exists after one of our older permission refactors.